### ISSUE TITLE
Calling GET /areas with no params returns 500

### ISSUE THREAD

**Issue by yamac-kurtulus** [acikkaynak/deprem-yardim-backend#109]
Calling GET /areas with no params returns 500, it should return 400. It is a weird case as it throws the correct exception.

**Comment by OmerFI** [comment]
It may be because of raise statements in https://github.com/acikkaynak/deprem-yardim-backend/blob/main/applications/feeds/views.py#L60-L74

**Comment by yamac-kurtulus** [comment]
> It may be because of raise statements in https://github.com/acikkaynak/deprem-yardim-backend/blob/main/applications/feeds/views.py#L60-L74

Yep, but those are proper validation errors, so they should be collected by DRF. I'll take a look in the coming hours.
